### PR TITLE
Trivially upgrade to Python 3

### PR DESCRIPTION
**Pull request by Vivraan**
I'm attempting to make use of Python 3 syntax with as much compatibility as possible with Python 3.4.

I incidentally use this fork of the directory on my Android phone for browing the book.

### CODE REVIEW

File: src/parseBook.py [hunk 1]
```diff
@@ -46,7 +44,7 @@ def injectShaderBlocks(_folder, _text):
             shaderCommand = "glslViewer " + shaderPath + " " + \
                             " ".join(shaderTexturePaths) + \
                             " -s 0.5 --headless -o " + shaderImage
-            print shaderCommand
+            print(shaderCommand
```
Comment (xav-ie) [inline src/parseBook.py:47]: I think you forgot a right parentheses?

File: src/parseBook.py [hunk 2]
```diff
@@ -55,7 +53,7 @@ def injectShaderBlocks(_folder, _text):
             gifName, gifExt = os.path.splitext(gifPath)
             pngImage = gifName + ".png"
             convertCommand = "convert " + gifPath + " " + pngImage
-            print convertCommand
+            print(convertCommand
```
Comment (xav-ie) [inline src/parseBook.py:56]: Also need closing parentheses here.. sorry if this is wrong!

Comment (Vivraan) [reply src/parseBook.py:56]: Indeed, I'd forgotten these. Fixing.

**Comment by patriciogonzalezvivo** [comment]
Thanks

**Comment by Vivraan** [comment]
@patriciogonzalezvivo @xav-ie 
**NOTE** aLL READMEs should be updated to mention that the required version is Python 3.

**Comment by Vivraan** [comment]
New PR here:
https://github.com/patriciogonzalezvivo/thebookofshaders/pull/317

**Pull request merged**